app/models/subject.rb

```diff
@@ -139,9 +139,9 @@ def download_comments
 
   def github_client
     if app_installation.present?
-      Octobox.installation_client(app_installation.github_id)
+      app_installation.github_client
     else
-      users.first&.subject_client
+      users.first&.github_client
```

Comment:
This is a change in logic, but I believe if we already know the `app_installation` is not present for this Repo, we should NOT try to use the app access token

Reply:
👍